peak/assembler/assembled_adt.py

```diff
@@ -159,11 +159,21 @@ def __getitem__(cls, key: tp.Tuple[BoundMeta, AssemblerMeta, AbstractBitVectorMe
             return cls.unbound_t[(val, *cls.fields[1:])]
         else:
             adt_t = key[0]
-            if (_issubclass(adt_t, AbstractBitVector)
-                or _issubclass(adt_t, AbstractBit)):
-                # Bit of a hack but don't bother wrapping Bits/Bitvectors
-                # Removes the issue of adding __operators__
-                return adt_t
+            fam = key[2].get_family()
```

Comment:
@rdaly525 note this change.  Basically makes `AssembledADT.bitvector_field` return the specified bv_type not the type in the ADT